mscore/resourceManager.cpp

```diff
@@ -42,9 +75,20 @@ ResourceManager::ResourceManager(QWidget *parent) :
       extensionsTable->verticalHeader()->hide();
       extensionsTable->setColumnWidth(1, 50);
       extensionsTable->setColumnWidth(1, 100);
+      pluginsTable->horizontalHeader()->setSectionResizeMode(QHeaderView::Stretch);
+      pluginsTable->verticalHeader()->hide();
       MuseScore::restoreGeometry(this);
       }
 
+//---------------------------------------------------------
+//   accept
+//---------------------------------------------------------
+void ResourceManager::closeEvent(QCloseEvent *e)
```

Comment:
Causes a compiler warning "warning C4100: 'e': unreferenced formal parameter". So add a `Q_UNUSED(e);` into the function